connectors/uipath/element-templates/uipath-custom-template.json

```diff
@@ -0,0 +1,575 @@
+        "oneOf": [
+          "getItemStatus"
+        ]
+      }
+    },
+    {
+      "label": "Organization Unit Id",
+      "group": "configuration",
+      "description": "UIPath Folder/OrganizationUnit Id",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "organizationUnitId"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Headers",
+      "description": "Map of HTTP headers to add to the request",
+      "group": "configuration",
+      "type": "Hidden",
+      "value": "={\"X-UIPATH-OrganizationUnitId\":organizationUnitId, \"Content-Type\":\"application/json\"}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "headers"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Queue Name",
+      "description": "The name of the queue to which to submit the queue item",
+      "group": "input",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "queueName0"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "addQueueItem"
+        ]
+      }
+    },
+    {
+      "label": "DeferDate",
+      "description": "The earliest date and time at which the item is available for processing. If empty the item can be processed as soon as possible",
+      "group": "input",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "deferDate0"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "addQueueItem"
+        ]
+      },
+      "optional": true
+    },
+    {
+      "label": "DueDate",
+      "description": "The latest date and time at which the item should be processed. If empty the item can be processed at any given time",
+      "group": "input",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "dueDate0"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "addQueueItem"
+        ]
+      },
+      "optional": true
+    },
+    {
+      "label": "Priority",
+      "id": "priority",
+      "group": "input",
+      "description": "The priority to apply to the queue item.",
+      "value": "Low",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Low",
+          "value": "Low"
+        },
+        {
+          "name": "Normal",
+          "value": "Normal"
+        },
+        {
+          "name": "High",
+          "value": "High"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "priority0"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "addQueueItem"
+        ]
+      }
+    },
+    {
+      "label": "Specific Content for UIPath Job",
```

Comment:
for me it was just working with an feel expression. if it is like this, then set feel as required